consensus/replay.go

```diff
@@ -320,6 +326,7 @@ func (h *Handshaker) replayBlocks(proxyApp proxy.AppConns, appBlockHeight, store
 func (h *Handshaker) replayLastBlock(proxyApp proxy.AppConnConsensus) ([]byte, error) {
 	mempool := types.MockMempool{}
 	cs := NewConsensusState(h.config, h.state, proxyApp, h.store, mempool)
+	defer cs.Stop()
```

Comment:
Indeed. Though I've seen it before as well. I think in this case, where `Start` may start a go-routine and then return an error, it makes sense to put the `defer Stop` before, otherwise we would need to have it twice - once in the error handling and once outside it.